src/v2/Apps/FairOrginizer/__tests__/FairOrganizerApp.jest.tsx

```diff
@@ -21,14 +26,18 @@ const { getWrapper } = setupTestWrapper<FairOrganizerApp_Test_Query>({
 
 describe("FairOrganizerApp", () => {
   it("sets a title tag", () => {
+    const profile = {
+      image: { url: "https://randomhash.cloudfront.net/12312/test.jpg" },
+    }
```

Comment:
hm, do we need this?

Reply:
we don't, put it in order to be more close to the reality, can remove if you ask to 😬